eclair-core/src/test/scala/fr/acinq/eclair/db/AuditDbSpec.scala

```diff
@@ -458,17 +458,15 @@ class AuditDbSpec extends AnyFunSuite {
             }
           },
           dbName = "audit",
-          targetVersion = 6,
+          targetVersion = 7,
           postCheck = connection => {
             val migratedDb = dbs.audit
-            using(connection.createStatement()) { statement =>
-              assert(getVersion(statement, "audit").contains(6))
-            }
+
```

Comment:
Is it necessary to remove that check?

Reply:
The `migrationCheck` does this exact test as part of the normal testing.